tests/test_inside_gha.sh

```diff
@@ -5,6 +5,10 @@ OSP_TOKEN_PATH=/tmp/token
 COMMON_DOCKER_ARGS="run --user osg
                         --detach
                         --security-opt apparmor=unconfined
+                        --security-opt seccomp=unconfined
+                        --security-opt systempaths=unconfined
+                        --security-opt no-new-privileges
+                        --net=host
```

Comment:
Switching to host networking is a big change; is that something we're doing just for GHA or are we going to be telling people to do it for their pilots?

Reply:
We started doing it here https://github.com/opensciencegrid/docs/pull/874

Reply:
Okay, then.
Might want to explain in that document why you are doing it but that's a separate PR.